docs/getting-started/new-project-guide/go_lang.md

```diff
@@ -22,13 +22,18 @@ process. The key specifics of integrating a Go project are outlined below.
 ## Go-fuzz support
 
 OSS-Fuzz supports **go-fuzz** in the
-[libFuzzer compatible mode](https://github.com/dvyukov/go-fuzz#libfuzzer-support)
+[libFuzzer compatible mode](https://github.com/mdempsky/go114-fuzz-build)
 only. In that mode, fuzz targets for Go use the libFuzzer engine with native Go
 coverage instrumentation. Binaries compiled in this mode provide the same
 libFuzzer command line interface as non-Go fuzz targets.
 
 ## Project files
 
+First, you need to write a Go fuzz target that accepts a stream of
+random bytes and calls the program API with that. This fuzz target
```

Comment:
nit: let's remove "random", I think it might be misleading

Reply:
Done!